### DISCUSSION TITLE
Live datas and bargraphs

### DISCUSSION THREAD

**Discussion by Jei23** [snipem/gt7dashboard#15] [Ideas]
Hello!
I was thinking that would be nice to have a "Live Data" tab where we can see live data and live animated graphics, same as in the "get faster" tab, but real time, for the current lap.
Could it be possible to implement that?
It could be interesting to put those in an overlay over a stream :)

**Comment by snipem** [comment]
Great idea. Problem is that Bokeh is already under high load when rendering the laps. You see that loading a lap takes 1-2 seconds, depending on your computer. It could work, but it should be separated from gt7dashboard, to reduce the stress on the already existing graphics.

**Reply by Jei23** [reply]
I've tried to do something, but, even if I can visualize real time datas (eg, speed, shift, bar graph with throttle and brake), I can't visualize real time "bar graphics" to show "real time telemetry", I'm having troubles in populating the graphics itself.
Let say that I'm not a programmer, I lack the basics, so maybe I'm just doing thing wrong :P
This is what I've done
![immagine](https://github.com/snipem/gt7dashboard/assets/114442648/de01b3b3-77cf-4288-8e18-d4313a00d8ec)
As you can see I've added a "Realtime Data" tab, in which I show the data you see.
The 4 green circles are the spin for the 4 tires.
I visualize those data in a 60hz refresh rate (16.67ms), and there are no problems at all.
Maybe I'll do some other test in the next days, but for now I'm stucked here :P.
Thanks for the amazing work, anyway!
